### PR TITLE
OO#96 when changing abotyp name this change is spread out

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala [hunk 1]
```diff
@@ -921,4 +940,13 @@ class StammdatenUpdateService(override val sysConfig: SystemConfig) extends Even
       }
     }
   }
+
+  private def getUpdatedDescriptionLieferplanung(abotypDepotTour: String, newName: String, oldName: String): String = {
+    val abotypDepotTourReplaced = abotypDepotTour
+      .replaceAll(oldName + ',', newName)
```
Comment (AlwinEgger) [inline src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala]: here the `oldname,` is replaces by `newName` (no comma). Same line below.

File: src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala [hunk 2]
```diff
@@ -921,4 +940,13 @@ class StammdatenUpdateService(override val sysConfig: SystemConfig) extends Even
       }
     }
   }
+
+  private def getUpdatedDescriptionLieferplanung(abotypDepotTour: String, newName: String, oldName: String): String = {
```
Comment (AlwinEgger) [inline src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala]: This logic must have been somewhere beforehand, no?

File: src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala (hunk 1 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala]: you're totally right. Changed it

File: src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala (hunk 2 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/stammdaten/StammdatenUpdateService.scala]: There are some helper functions in this service that is why I decided to place this function here (f.i. adjustGuthabenVorLieferung). However I can create a handler trait for the lieferplanung. this is what you meant??

**Comment by ClSchreiber** [comment]
soliterre-Test: funktioniert perfekt! Grazie mille

**Pull request merged**